_config.yml

```diff
@@ -8,6 +8,8 @@ languages:
   - en
   - es
   - fr
+# Do not show Chinese in dropdown yet
+# - zh
```

Comment:
This doesn't prevent it from being included in the sitemap and therefore more likely to be crawled & surfaced in search results. Per my previous point, I wonder if we could prevent the pages from being compiled at all in the live site or at least exclude from sitemap until we're ready?

https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/mitchellhenke/chinese-translations/sitemap.xml

Reply:
I _think_ the primary option there would be `sitemap: false` in the front matter, which isn't ideal?